apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -328,6 +329,17 @@ export default Vue.extend({
           label: '<x-menuitem><x-label>Copy Row (TSV / Excel)</x-label></x-menuitem>',
           action: (_e, cell) => this.$native.clipboard.writeText(Papa.unparse([cell.getRow().getData()], { header: false, delimiter: "\t", quotes: true, escapeFormulae: true }))
         },
+        {
+          label: '<x-menuitem><x-label>Copy as Insert</x-label></x-menuitem>',
+          action: async (_e, cell) => {
+            const exporter = new SqlExporter("./tmp/sql.export", this.connection, this.table, [], { chunkSize: 100 }, {})
+            await exporter.exportToFile();
+
+            const cells = cell.getRow().getCells().map(cell => cell.getValue())
+            const formatted = exporter.formatRow(cells)
```

Comment:
This might be the simplest way I found to generate an insert query, though it still exports a new file as a side effect.

I'm thinking of extending the Exporter so that probably we can do `new SqlExporter(this.connection, this.table, ...).getExportString()`? What do you think about this problem @rathboma ?